rcp/org.testeditor.rcp4.tcltestrun/src/org/testeditor/rcp4/tcltestrun/TclGradleLauncher.xtend

```diff
@@ -43,7 +43,8 @@ public class TclGradleLauncher implements TclLauncher {
 		val resultHandler = new TclGradleResultHandler(testResultFolder)
 		gradleHelper.run(project.location.toFile, resultHandler) [
 			if (testCases !== null) {
-				withArguments("clean", "test", "--tests", testCase) // https://issues.gradle.org/browse/GRADLE-2972
+    			logger.info('''Testcase '«testCase»' is running.''')
+    			withArguments("clean", "test", "--tests" , testCase ) // https://issues.gradle.org/browse/GRADLE-2972
 			} else {
 				withArguments("clean", "test")
```

Comment:
how about (if the above is logged, this should be logged, too)
logger.info('''Gradle run configured with arguments 'clean test'.''')
?